doc/sphinx/Pacemaker_Remote/baremetal-tutorial.rst

```diff
@@ -72,7 +72,7 @@ node.
 .. code-block:: none
 
     # cat << END >> /etc/hosts
-    192.168.122.10  remote1
+    192.168.122.103  remote1
```

Comment:
I'm not sure the IP address changes belong in this commit -- are they related to the OS switch?

BTW the rest of this commit makes sense as a single commit since everything is related and necessary for it to work

Reply:
Nah, I'll remove.